setup/authanywhere.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+if [ "$(uname -m)" == "x86_64" ]; then
+    authanywhere_arch="amd64"
+elif [ "$(uname -m)" == "aarch64" ]; then
+    authanywhere_arch="arm64"
+else
+    echo "Unsupported architecture to install authanywhere: $(uname -m)"
+    exit 1
+fi
+
+# Install authanywhere for infra token management
+curl -OL "binaries.ddbuild.io/dd-source/authanywhere/LATEST/authanywhere-linux-$authanywhere_arch" && \
+    mv "authanywhere-linux-$authanywhere_arch" /bin/authanywhere && \
+    chmod +x /bin/authanywhere
```

Comment:
```suggestion
curl -OL "binaries.ddbuild.io/dd-source/authanywhere/LATEST/authanywhere-linux-$authanywhere_arch"
mv "authanywhere-linux-$authanywhere_arch" /usr/local/bin/authanywhere
chmod +x /usr/local//bin/authanywhere
```
- no longer being in a Dockerfile, the chain of `&&` is no longer needed (and we have the fail-fast thanks to `-e`)
- prefer `/usr/local/bin` over `/bin`:
    - `/bin` is deprecated on modern Linux distros (either a symlink to `/usr/bin` or even absent)
    - for packages installed out of package manager control (FHS)
    - also because of precedence in "canonical" `PATH`s (usually `...:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:...`)